File: tests/stew-gettext/c++-ngettext/source.cpp
#include <libintl.h>
#include <locale.h>

int main (int argc, char **argv)
{
    int n = 5;
    char *ng1 = ngettext("ngettext1", "ngettext1-plural", n);
    char *dng1 = dngettext("test-domain", "dngettext1", "dngettext1-plural", n);
    char *dng2 = dngettext("some-other-domain", "dngettext2", "dngettext2-plural", n);
    char *dcng1 = dcngettext("test-domain", "dcngettext1", "dcngettext1-plural", n, LC_MESSAGES);
    char *dcng2 = dcngettext("some-other-domain", "dcngettext2", "dcngettext2-plural", n, LC_MESSAGES);

    return 0;
}

File: tests/stew-gettext/python-ngettext/source.py
import gettext

n = 5
ng1 = gettext.ngettext("ngettext1", "ngettext1-plural", n)
dng1 = gettext.dngettext("test-domain", "dngettext1", "dngettext1-plural", n)
dng2 = gettext.dngettext("some-other-domain", "dngettext2", "dngettext2-plural", n)
